.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint PR
+
+on:
+  pull_request_target:
+    types: [opened, edited, synchronize]
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Lint pr title
+        uses: amannn/action-semantic-pull-request@v3.4.0
+        with:
+          wip: true
+          types: |
```

Comment:
Are these types documented somewhere?

Reply:
no, at least not in this repo not yet. This list was taken from the defaults at https://github.com/trussworks/react-uswds/blob/main/docs/contributing.md#type and we also added `component`. I think we are still deciding on what the "final" list should be since our release strategy is still kind of up in the air

Reply:
Can we bring that over?

Reply:
Yes! I updated the README with some basics and the descriptions of these